### ISSUE TITLE
Issue building Samba4

### ISSUE THREAD

**Issue by nidstigator** [Andy2244/openwrt-extra#1]
Hey Andy! 

Many thanks for your work! I tried to build Samba4 using the instructions in your README; while pulling the latest available source from LEDE (master branch). I get the following error when attempting to build samba4:

ERROR: glib-2.22 gthread-2.0 is required to compile QEMU

Any ideas?

Thanks!

**Comment by Andy2244** [comment]
Its always possible that something breaks via master, since i still wait for the actual merge request to-be accepted and i don't have setup any CI.

Seem to me that they split gthread or changed the hostbuild behavior. Will try to fix this if i have some spare time.

**Comment by nidstigator** [comment]
Andy, thanks for responding! 
I actually tried to build it also against the latest ‘stable’ LEDE branch. The samba4 package itself this time fails to build with : 
ERROR: Target 'com_err' re-defined as SYSLIB

Have you seen this one before?

**Comment by Andy2244** [comment]
@nidstigator Ok, something is strange i cant reproduce the error using the latest master. What's strange "ERROR: glib-2.22 gthread-2.0 is required to compile QEMU" actually qemu should try compile glib-2.5x not 2.22 ?

Against what master/head you are compiling? 
`git rev-parse HEAD`

What distro and .config you are using?
`./scripts/diffconfig.sh > my.diff`
(post the my.diff)

PS: You cant compile against the stable branch, its incompatible with it.

**Comment by nidstigator** [comment]
Andy, actually, this is issue was totally due to my own wrongdoing. I was trying to compile the package accidently against a different LEDE branch that I had cloned a long time ago, not the one provided by official LEDE git.

I managed to compile samba4 and all of its dependencies. On a side note; libcomerr conflicted with a package I already had installed; libext2fs. They both try to install a com_err.so.0 file 

Thanks for the effort!
